diff --git a/node_modules/@noble/hashes/utils.js b/node_modules/@noble/hashes/utils.js
index c0519ac..4580d7e 100644
--- a/node_modules/@noble/hashes/utils.js
+++ b/node_modules/@noble/hashes/utils.js
@@ -154,8 +154,14 @@ exports.wrapConstructorWithOpts = wrapConstructorWithOpts;
 function randomBytes(bytesLength = 32) {
     if (crypto_1.crypto && typeof crypto_1.crypto.getRandomValues === 'function') {
         return crypto_1.crypto.getRandomValues(new Uint8Array(bytesLength));
+	}
+	try {
+		const randomBytes = require('expo-crypto').getRandomBytes;
+        return randomBytes(bytesLength);
+    } catch (e) {
+        throw new Error("The environment doesn't have randomBytes function and we couldn't use expo-crypto");
     }
-    throw new Error('crypto.getRandomValues must be defined');
+    // throw new Error('crypto.getRandomValues must be defined');
 }
 exports.randomBytes = randomBytes;
 //# sourceMappingURL=utils.js.map
\ No newline at end of file
